suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -784,23 +784,16 @@ export const selectDeviceFirmwareVersion = memoize((state: DeviceRootState) => {
 
     return getFirmwareVersionArray(device);
 });
-
-export const selectPersistedDeviceStates = (state: DeviceRootState) => {
-    const devices = selectDevices(state);
-
-    return [...devices.map(d => d.state), PORTFOLIO_TRACKER_DEVICE_STATE];
-};
-
 export const selectPhysicalDevices = memoize((state: DeviceRootState) => {
     const devices = selectDevices(state);
 
     return devices.filter(device => device.id !== PORTFOLIO_TRACKER_DEVICE_ID);
 });
 
 export const selectIsNoPhysicalDeviceConnected = (state: DeviceRootState) => {
-    const devices = selectDevices(state);
+    const devices = selectPhysicalDevices(state);
 
-    return devices.length === 1 && devices[0].id === PORTFOLIO_TRACKER_DEVICE_ID;
+    return !devices.filter(device => device.connected).length;
```

Comment:
For me Array.some would be better for readability, but it's just a nit

Reply:
8a8749319f375d0330bf6d462b88755f13445153

Reply:
Even better 👏